Library/Homebrew/livecheck/strategy/github_latest.rb

```diff
@@ -0,0 +1,59 @@
+# typed: false
+# frozen_string_literal: true
+
+module Homebrew
+  module Livecheck
+    module Strategy
+      # The {GithubLatest} strategy identifies versions of software at
+      # github.com by checking a repository's latest release page.
+      #
+      # GitHub URLs take a few differemt formats:
+      #
+      # * `https://github.com/example/example/releases/download/1.2.3/example-1.2.3.zip`
+      # * `https://github.com/example/example/archive/v1.2.3.tar.gz`
+      #
+      # This strategy is used when latest releases are marked for software hosted
+      # on GitHub. It is necessary to use `strategy :github_latest` in a `livecheck`
+      # block for Livecheck to use this strategy.
+      #
+      # The default regex identifies versions from `href` attributes containing the
+      # tag name.
+      #
+      # @api public
+      class GithubLatest
```

Comment:
I'm OK with this name but wonder if `GitHub` or `GitHubRelease` would suit it better.

Reply:
I wanted to explicitly state that we're using the latest release, so I went with this one. There may be other better names though. `GithubLatestRelease` was too long, but if that's preferable I'll be happy to change it.